slackviewer/main.py

```diff
@@ -33,7 +33,7 @@ def configure_app(app, archive, debug):
 @click.option("-z", "--archive", type=click.Path(), required=True,
               default=envvar('SEV_ARCHIVE', ''),
               help="Path to your Slack export archive (.zip file or directory)")
-@click.option('-I', '--ip', default=envvar('SEV_IP', 'localhost'),
+@click.option('-I', '--ip', default=envvar('SEV_IP', '0.0.0.0'),
```

Comment:
No need to change the default. Just do `export SEV_IP=0.0.0.0` in your shell.

Reply:
done